prow/config/config.go

```diff
@@ -666,6 +670,14 @@ type GitHubOptions struct {
 	LinkURL *url.URL `json:"-"`
 }
 
+// ManagedHmacEntityInfo contains information about the repo/org which is onboarded and associated metadata.
+type ManagedHmacEntityInfo struct {
+	TokenCreatedAfter time.Time `json:"tokenCreatedAfter"`
```

Comment:
~~FYI, if we use `time.Time` here instead of a string that we deserialize ourselves this will be the time format:
`2009-11-10T23:00:00Z` https://play.golang.org/p/lpWRnIuWZ-X~~

Just saw that matches your DD so 👍